testsuite/smoke-test/src/inspection_service.rs

```diff
@@ -0,0 +1,16 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::smoke_test_environment::new_local_swarm_with_aptos;
+use aptos_forge::Swarm;
+
+#[tokio::test]
+async fn test_inspection_service_connection() {
+    let mut swarm = new_local_swarm_with_aptos(1).await;
+    let info = swarm.aptos_public_info();
+
+    let resp = reqwest::get(info.inspection_service_url().to_owned())
```

Comment:
nit: can we add a comment to this? e.g.,:

```
// Ping the inspection service index page and verify we get a successful response
```

Reply:
Done.